recipes/alerter.go

```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```

Comment:
**Should** we ignore this in the build? It seems like a good idea to at least make sure this builds in CI.

Here and below.

Reply:
Those two file only had the old build tags but as you recently bumped the Go version to 1.21 I had to add the newer build tags as gopls otherwise complained for these two files. It was previously ignored to be built but I don't know the rationale behind it.

Reply:
No rationale, TBH we just haven't had a good maintenace story for this repo. (This code was originally written before the GitHub Actions product was generally available!)